### PR TITLE
NuCivic/internal#783 Move title styles from .js to .css file

### PR DESCRIPTION
**Pull request by janette**
ref NuCivic/internal#783

### CODE REVIEW

**Comment by nucivicmachine** [comment]
QA site created: http://783-ve-updates.visualization_entity_charts.qa.nuamsdev.com

**Comment by nucivicmachine** [comment]
QA site created: http://783-ve-updates.visualization_entity_charts.qa.nuamsdev.com

File: js/visualization_entity_charts_steps.js [hunk 1]
```diff
@@ -355,10 +355,10 @@ this.recline.View.nvd3 = this.recline.View.nvd3 || {};
         backend: backend,
         url: url
       };
-      
+
       state.set('source', source);
       $.get(url).done(function(data){
-        data = data.replace(/(?:\r|\n)/g, '\r\n');
+        //data = data.replace(/(?:\r|\n)/g, '\r\n');
```
Comment (acouch) [inline js/visualization_entity_charts_steps.js]: Why is this getting removed?

**Comment by nucivicmachine** [comment]
QA site created: http://783-ve-updates.visualization_entity_charts.qa.nuamsdev.com

**Comment by topicus** [comment]
This became old. The library changed a lot since this PR was made. Closing this.

**Pull request closed without merging**